### PR TITLE
update docs/examples, use ci templates and mysql

### PR DESCRIPTION
**Pull request by weshayutin**
* Previously we had duplicate templates for mssql
* Templates used in docs/examples should be part of
regular ci to ensure they always work for the community
  * using softlinks from tests/e2e to docs/examples/
* update the docs s/mssql/mysql

Closes-Issue: https://github.com/openshift/oadp-operator/issues/622

### CODE REVIEW

**Comment by weshayutin** [comment]
@deepak1725 @kaovilai @hhpatel14  @EmilyM1 FYI.. I have updated or changed some of your doc or templates. Please review.  Thank you!

File: docs/examples/manifests/mysql [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/whayutin/OPENSHIFT/git/OADP/oadp-operator/tests/e2e/sample-applications/mysql-persistent/
```
Comment (kaovilai) [inline docs/examples/manifests/mysql:1]: What?

File: docs/examples/manifests/nginx [hunk 2]
```diff
@@ -0,0 +1 @@
+/home/whayutin/OPENSHIFT/git/OADP/oadp-operator/tests/e2e/sample-applications/nginx/
```
Comment (kaovilai) [inline docs/examples/manifests/nginx:1]: What are these for?

**Comment by deepak1725** [comment]
> @deepak1725 @kaovilai @hhpatel14 @EmilyM1 FYI.. I have updated or changed some of your doc or templates. Please review. Thank you!

I am not sure why I am tagged here.

File: docs/examples/manifests/mysql (hunk 1 above)
Comment (weshayutin) [reply docs/examples/manifests/mysql:1]: crap.. soft links were not relative

**Comment by kaovilai** [comment]
I was like damn.. that new profile pic.

**Comment by hhpatel14** [comment]
LGTM

**Review by dymurray — APPROVED** [review]

**Pull request merged**